### ISSUE TITLE
Add a different SFx for team flag...

### ISSUE THREAD

**Issue by Punjara** [assaultcube/AC#284]
It would be nice if we get an audio feedback rather than looking on the screen if the team flag is taken, so that player could really focus on enemy (it would be more immersive). So it would be nice if we get a different SFx for team flag taken or restored.

**Comment by cf-1984** [comment]
Good idea, certainly would help, especially because there is no visual indication which team's flag was taken during death-screen/respawn cooldown (when 'extra messages' are disabled).

Would it require an entirely different sound effect, or would an altered version (changed pitch, slightly shorter/longer, brighter/darker) do the trick?

**Comment by Punjara** [comment]
I mean you can do altered version to save time, but idea is to recognize it better. If certain audio properties are same as the normal SFX, then even it is altered, brain would be confused, because reaction time for this in the middle of the fight is in milliseconds. People can miss the audio que completely if they are in fight if duration is too small, because our brain focus on visual senses more during these fights, or Team flag and Enemy flags are taken at the same (altered and original would mask each other). So it should not be continues, rather it should be repeated throughout or at least ring three times like Amplitude Modulation (square wave) "a Siren like SFx' so that player wouldn't miss it. It shouldn't be harsh that it would annoy or neither should it be very mild so that it would becomes a lullaby.
